debian/changelog

```diff
@@ -1,3 +1,10 @@
+ureadahead (0.100.1-0.1) unstable; urgency=low
```

Comment:
what's with the decimal "0.1" versioning? AFAICT, this versioning format doesn't really need decimals for the last component. Just positive integers.

Reply:
This doesn't break anything on my end. Should I wait for an update or just pull this as is?

Reply:
eh, my eyes just start glazing over when there are so many irrelevant numbers. merge or fix, I don't care that much.

Reply:
Thanks, I'll just merge it.